tests/expressions/tests.py

```diff
@@ -1055,6 +1079,46 @@ def test_update_UUIDField_using_Value(self):
         UUID.objects.update(uuid=Value(uuid.UUID('12345678901234567890123456789012'), output_field=UUIDField()))
         self.assertEqual(UUID.objects.get().uuid, uuid.UUID('12345678901234567890123456789012'))
 
+    def test_deconstruct_Value(self):
+        value = Value('name')
+
+        path, args, kwargs = value.deconstruct()
+
+        self.assertEqual(path, 'django.db.models.expressions.Value')
+        self.assertEqual(args, (value.value,))
+        self.assertEqual(kwargs, {})
+
+    def test_deconstruct_Value_output_field(self):
+        value = Value('name', output_field=CharField())
+
+        path, args, kwargs = value.deconstruct()
+
+        self.assertEqual(path, 'django.db.models.expressions.Value')
+        self.assertEqual(args, (value.value,))
+        self.assertEqual(len(kwargs), 1)
+        self.assertEqual(
```

Comment:
I'd use a single line here. We allow lines up to 119 chars when it helps readability.